checkpatch_hook/data/checkpatch.yaml

```diff
@@ -0,0 +1,47 @@
+ERRORS_COMMON:
+  - TRAILING_WHITESPACE
+  - COMPLEX_MACRO
+  - DOS_LINE_ENDINGS
+  - DO_WHILE_MACRO_WITH_TRAILING_SEMICOLON
+  - EXECUTE_PERMISSIONS
+  - INVALID_UTF8
+  - LINE_CONTINUATIONS
+  - MACRO_ARG_PRECEDENCE
+  - MASK_THEN_SHIFT
+  - MISSING_EOF_NEWLINE
+  - MULTISTATEMENT_MACRO_USE_DO_WHILE
+  - ONE_SEMICOLON
+  - BRACKET_SPACE
+  - TYPO_SPELLING
+
+IGNORES_COMMON:
+  - BAD_SIGN_OFF
+  - SPDX_LICENSE_TAG
+  - FILE_PATH_CHANGES
+  - NOT_UNIFIED_DIFF
+  - LINUX_VERSION_CODE
+  - CONSTANT_COMPARISON
+  - OPEN_ENDED_LINE
+  - UNNECESSARY_PARENTHESES
+  - GERRIT_CHANGE_ID
+  - COMMIT_LOG_LONG_LINE
+  - EMAIL_SUBJECT
+  - GIT_COMMIT_ID
+
+# Per-directory checkpatch configuration
+DIR_CONFIGS:
+  default:
```

Comment:
Hmm how about `__default__` to really avoid a potential name collision and because double underscores are very Pythonic.